admin/admin-service/src/main/java/io/mosip/admin/service/impl/AdminServiceImpl.java

```diff
@@ -20,12 +20,18 @@
 
 @Service
 public class AdminServiceImpl implements AdminService {
+	
+	private String LOST_RID_DATE_RANGE_ERROR_CODE = "RPR-RGS-034";
+	private String LOST_RID_DATE_RANGE_ERROR_MSG = "searching between date should be less then %s days";
```

Comment:
Why is the message hardcoded in the service class?

Reply:
I reverted that commit in pr as the change in dynamic field controller is urgent. I'll raise a new pr with the changes that you suggested.